numpy/core/fromnumeric.py

```diff
@@ -782,7 +782,7 @@ def argpartition(a, kth, axis=-1, kind='introselect', order=None):
     axis : int or None, optional
         Axis along which to sort. The default is -1 (the last axis). If
         None, the flattened array is used.
-    kind : {'introselect'}, optional
+    kind : {'introselect','floydselect'}, optional
```

Comment:
```suggestion
    kind : {'introselect', 'floydselect'}, optional
```

Reply:
The default of "introselect" should stay the same?  If floydselect is almost always faster, we could consider changing the default.

I also think this change deserves a mention in the release note as an enhancement or performance, pick the one you prefer.